lib/msf/core/exploit/remote/socket_server.rb

```diff
@@ -26,6 +26,8 @@ def initialize(info = {})
 
     register_advanced_options(
       [
+        OptAddress.new('ListenerBindAddress', [ false, 'The specific IP address to bind to if different from SRVHOST']),
```

Comment:
I wish I could think of a better name for this, but I think `ListenerBindAddress` may end up being too generic if we want to add future bind address concepts in the future. Potentially `SrvReverseBindAddress` ?